src/plugins/stringStyle.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+import createRule from '../utils/createRule'
+import type {JssStyle, RuleOptions, Rule} from '../types'
+
+const nlOrSemi = /\n|;/
```

Comment:
Depending on how far you want to go to make sure no semicolons work, this will fail with valid input like.

```css
box-shadow: 0, 0, 0 #fff,
                       0, 0, 0 #ccc   ,    
                       0, 0, 0 #000
```

Though this might help there.

```js
/[^,]\s*\n|;/
```